### DISCUSSION TITLE
Field "reporter" for project views

### DISCUSSION THREAD

**Discussion by DKhalil** [community/community#46868] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

It would be very useful to have the issue reporter as an available field for the project views, especially in cases where the reporter would want to have a separate view of all open issues they reported.

**Comment by queenofcorgis** [comment]
Hi there! 

Thanks for sharing this feedback 🌟 . To help us better understand your request, would you mind sharing what you mean by issue reporter.. do you mean the assignee?

**Reply by DKhalil** [reply]
No, I meant the one who wrote (reported) the issue

**Reply by queenofcorgis** [reply]
Thanks for clarifying, so the author if I'm understanding correctly now?

**Reply by DKhalil** [reply]
Yes
